### DISCUSSION TITLE
Equiaxed microstructure with a defined number of grains (n = 250 grains for example)

### DISCUSSION THREAD

**Discussion by amgamo1** [neperfepx/neper#360] [Q&A] [answered]
We're trying to generate an equiaxed microstructure with a set number of grains 250. The grain growth morphology with command "-morpho gg" does not provide the equiaxed structure we need.  We're able to generate the equiaxed microstructure by specifying the our own grain size distribution values following the example  command:
neper -T -n from_morpho -morpho"diameq:lognormal(0.1,0.03),1-sphericity:lognormal(0.145,0.03) 

However, with this method, we can't set the grain size to a specific number. Is there a method to generate a more equiaxed structure than "-morpho gg" with a specified number of grains?

Thank you

**Comment by rquey** [comment, marked as answer]
I'm not sure I understand. By "more equiaxed", do you mean "grains with more similar volumes"? In that case, just reduce the standard deviation in the diameq lognormal.
